test/e2e/framework/enable.go

```diff
@@ -17,46 +17,28 @@ limitations under the License.
 package framework
 
 import (
-	"fmt"
-	"io/ioutil"
-	"path/filepath"
-	"runtime"
+	"bytes"
 	"strings"
 
 	kfenable "sigs.k8s.io/kubefed/pkg/kubefedctl/enable"
 	"sigs.k8s.io/kubefed/test/common"
 )
 
 func LoadEnableTypeDirectives(tl common.TestLogger) []*kfenable.EnableTypeDirective {
-	path := enableTypeDirectivesPath(tl)
-	files, err := ioutil.ReadDir(path)
-	if err != nil {
-		tl.Fatalf("Error reading EnableTypeDirective resources from path %q: %v", path, err)
-	}
 	enableTypeDirectives := []*kfenable.EnableTypeDirective{}
 	suffix := ".yaml"
-	for _, file := range files {
-		if !strings.HasSuffix(file.Name(), suffix) {
-			continue
-		}
-		filename := filepath.Join(path, file.Name())
-		obj := kfenable.NewEnableTypeDirective()
-		err := kfenable.DecodeYAMLFromFile(filename, obj)
-		if err != nil {
-			tl.Fatalf("Error loading EnableTypeDirective from file %q: %v", filename, err)
+	prefix := "config/enabletypedirectives/"
+	for _, file := range common.AssetNames() {
+		if strings.Contains(file, prefix) {
+			fixtureAsset := common.MustAsset(file)
+			obj := kfenable.NewEnableTypeDirective()
+			filename := strings.TrimSuffix(strings.TrimPrefix(file, prefix), suffix)
+			err := kfenable.DecodeYAML(bytes.NewBuffer(fixtureAsset), obj)
+			if err != nil {
+				tl.Fatalf("Error loading EnableTypeDirective from file %q: %v", filename, err)
```

Comment:
Does it make sense to say 'from this file' given that the data is being loaded from the binary?